services/director-v2/src/simcore_service_director_v2/utils/dask.py

```diff
@@ -353,6 +356,17 @@ def from_node_reqs_to_dask_resources(
     return dask_resources
 
 
+def check_scheduler_is_still_the_same(
```

Comment:
minor: typing is missing in a few functions in this module

Reply:
actually this is something I was used to in openGL. A check does not return anything (which is why there is no type annotation btw) but throws an exception if the condition is not met.

I do not like ```assert_...*``` because asserts are typically removed in release mode. I could go for ```raise_if...``` .
I do not want to write ```if test_*``` with return values because that clutters the code.